soh/src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -1508,8 +1508,14 @@ void KaleidoScope_DrawPages(PlayState* play, GraphicsContext* gfxCtx) {
                 gDPSetCombineMode(POLY_KAL_DISP++, G_CC_MODULATEIA, G_CC_MODULATEIA);
                 gDPSetPrimColor(POLY_KAL_DISP++, 0, 0, 255, 255, 255, pauseCtx->alpha);
 
-                POLY_KAL_DISP =
-                    KaleidoScope_QuadTextureIA8(POLY_KAL_DISP, sPromptChoiceTexs[gSaveContext.language][0], 48, 16, 12);
+                if (!gSaveContext.isBossRush) {
+                    POLY_KAL_DISP = KaleidoScope_QuadTextureIA8(
+                        POLY_KAL_DISP, sPromptChoiceTexs[gSaveContext.language][0], 48, 16, 12);
+                } else {
+                    // Show "No" twice in Boss Rush because the player can't save within it.
+                    POLY_KAL_DISP = KaleidoScope_QuadTextureIA8(
+                        POLY_KAL_DISP, sPromptChoiceTexs[gSaveContext.language][1], 48, 16, 12);
+                }
```

Comment:
Not gonna require it for this first revision of boss rush, but we should probably just make pressing B on the pause menu just close the menu rather than this hacky situation.

Reply:
Actually it looks like it does do that later on, maybe this change is no longer necessary?

Reply:
This is for the game over screen, not the save screen.

Reply:
Ah ok, and we aren't able to just skip it like with the pause menu? I seem to recall that from one of your showcase streams but my memory is fuzzy.

Reply:
Technically I could, but I don't want to kick users back to the title screen immediately after they die. So instead I figured I could make it into something that's also a little funny, where it shows "no" "no" on the "continue playing?" screen. I do already skip past the "Do you want to save?" screen inside the game over screen though.

Made a clip for it so you can see how it looks in game: https://clips.twitch.tv/ElatedCrepuscularLEDCharlieBitMe-wOcr3C4DNmIqlw5H